fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Optimizer.java

```diff
@@ -219,4 +201,22 @@ private void collectAllScanOperators(Memo memo, TaskContext rootTaskContext) {
         Utils.extractOlapScanOperator(tree.getGroupExpression(), list);
         rootTaskContext.setAllScanOperators(Collections.unmodifiableList(list));
     }
+
+    void ruleRewriteIterative(Memo memo, TaskContext rootTaskContext, RuleSetType ruleSetType) {
```

Comment:
Add some comments let other people know when to call `ruleRewriteIterative`, when should call `ruleRewriteOnlyOnce`

Reply:
OK